code/drasil-printers/lib/Language/Drasil/HTML/Helpers.hs

```diff
@@ -130,8 +131,13 @@ descWrap :: [String] -> Doc -> Doc -> Doc
 descWrap = flip (wrapGen Class "dt")
 
 -- | Helper for wrapping divisions or sections.
+refwrap' :: Maybe String -> Doc -> Doc -> Doc
+refwrap' a = flip (wrapGen Id tag) [""]
+  where
+    tag = fromMaybe "div" a
```

Comment:
`fromMaybe` is a Haskell design smell -- please use `maybe` instead.

Reply:
This function definition is also a bit confusing. What are the arguments given to it? "Wrapping" node type, body, ...?